aviatrix/data_source_aviatrix_firewall_instance_images.go

```diff
@@ -68,12 +62,16 @@ func dataSourceAviatrixFirewallInstanceImagesRead(d *schema.ResourceData, meta i
 	for _, image := range *firewallInstanceImages {
 		fI := make(map[string]interface{})
 		fI["firewall_image"] = image.Image
-		listVersion := image.Version
-		sort.Sort(sort.Reverse(byVersion(listVersion)))
-		fI["firewall_image_version"] = listVersion
-		listSize := image.Size
-		sort.Sort(sort.Reverse(bySize(listSize)))
-		fI["firewall_size"] = listSize
+		versionList := image.Version
+		sort.Slice(versionList, func(i, j int) bool {
+			return sortVersion(versionList, i, j)
+		})
+		fI["firewall_image_version"] = reverseArray(versionList)
```

Comment:
Is this reverse really needed? I think you can change how the sort works so that we don't need to reverse the list after. Probably swapping true and false in `compareImageSize` should work but you will need to test that.

Reply:
remove the reverseArray function and set the compare somatic version function from LessThan to GreaterThan